includes/functions-host-json-durabletask.md

```diff
@@ -11,29 +11,35 @@ ms.custom: include file
 
 Configuration settings for [Durable Functions](../articles/azure-functions/durable-functions-overview.md).
 
+> [!NOTE]
+> The below examples are for use with Azure Functions V2 or V3. If using Azure Functions V1, the "durableTask" section of the host.json should go in the root, rather than as a field under "extensions".
+
+
 ### Durable Functions 1.x
```

Comment:
I wonder if we should put the Durable Functions 2.x section on top since that's generally the version we want people using.